### DISCUSSION TITLE
Gesamtertrag auf Display: Einheit bzw Kommastelle anpassen

### DISCUSSION THREAD

**Discussion by daytona80** [tbnobody/OpenDTU#2093] [Ideas]
Hallo,
Würde gern von kWh auf MWh für den Gesamtertrag umstellen, da im 1Zoll Display zu wenig Platz für die Anzeige ist (aktuell eine Nachkommastelle, welche gern entfalle könnte). Ja, hab schon über 10MWh Ertrag und unter gleichbleibender Annahme, wären es in 10 Jahren über 100MWh, also noch eine Stelle mehr.
Könnte man hierfür nicht einen "Schalter" in der Oberfläche einbauen?

Danke und Gruß

**Comment by Tiese** [comment]
Sollte das nicht eigentlich in der Datei: Display_Graphic.cpp automatisch erfolgen?

An dieser Stelle...
`// Total production
const float wattsTotal = Datastore.getTotalAcYieldTotalEnabled();
auto const format = (wattsTotal >= 1000) ? i18n_yield_total_mwh : i18n_yield_total_kwh;
snprintf(_fmtText, sizeof(_fmtText), format[_display_language], wattsTotal);
printText(_fmtText, 2);`

Komisch nur, dass weiter oben folgendes steht...
`static const char* const i18n_yield_total_kwh[] = { "total: %.1f kWh", "Ges.: %.1f kWh", "total: %.1f kWh" };
static const char* const i18n_yield_total_mwh[] = { "total: %.0f kWh", "Ges.: %.0f kWh", "total: %.0f kWh" };`
--------------------------------------------------------------------------**^**-------------------**^**--------------------**^**
